bin/create-scxa-analytics-schema.sh

```diff
@@ -79,41 +138,39 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
 
 #############################################################################################
 # 1.2
-printf "\n\nDelete dynamic field rule factor_* "
+printf "\n\nDelete dynamic field rule factor_*"
 curl -X POST -H 'Content-type:application/json' --data-binary '{
   "delete-dynamic-field":
   {
     "name": "factor_*"
   }
 }' http://$HOST/solr/$CORE/schema
 # 1.3
-printf "\n\nCreate dynamic field rule factor_* (lowercase, multiValued) "
+printf "\n\nCreate dynamic field rule factor_* (lowercase)"
 curl -X POST -H 'Content-type:application/json' --data-binary '{
   "add-dynamic-field":
   {
     "name": "factor_*",
-    "type": "lowercase",
-    "multiValued": true
```

Comment:
I'm too ignorant on Solr to understand the implications of this, could you briefly explain why we don't need the multivalued here anymore? Just for my understanding, thanks.

Reply:
The implication is that the `factor_*` fields can contain only a single value. I don’t think this is significant in any way, apart from the correctness of the data modelling, since we can’t have more than one value for any sample characteristic or experimental factor. I always found it slightly misleading. It might also simplify some `jq` or queries in the code because we won’t need to unpack the value from an array.

Reply:
Ahh. but then doesn't this imply that this is not directly backward compatible? This would require then changes to the queries, right?

Reply:
We need to run the integration tests in the web app. Since we work with an alias we can always switch it back to `v5` and use `v6` only for the cell type wheel queries, but we’ll know for sure after we merge https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/173. It will also depend on how low-level SolrJ is; for instance, there are places that even with single values it returns a multimap in the response.